src/libsync/configfile.cpp

```diff
@@ -78,6 +78,9 @@ static const char newBigFolderSizeLimitC[] = "newBigFolderSizeLimit";
 static const char useNewBigFolderSizeLimitC[] = "useNewBigFolderSizeLimit";
 static const char confirmExternalStorageC[] = "confirmExternalStorage";
 
+static const char deltaSyncEnabledC[] = "DeltaSync/enabled";
```

Comment:
We should agree on a common name for all variables. Either delta everywhere or z everywhere? Maybe zsync because it would (hypothetically) allow us to use another delasync algorithm at some point.

Reply:
My thinking was use zsync for all "behind the scenes" naming so it's clear for a programmer/developer but use delta-sync in UI/Config so it's clear to end-users ...

Reply:
That thinking sounds good too